streaming/index.js

```diff
@@ -280,7 +261,7 @@ const startServer = async () => {
    * @param {string} message
    * @param {string} channel
    */
-  const onRedisMessage = (message, channel) => {
+  const onRedisMessage = (channel, message) => {
```

Comment:
Make sure you update the documentation comment above, I believe the lines need to be ordered correctly